### DISCUSSION TITLE
TheAgencyTheme (or any theme) change background image

### DISCUSSION THREAD

**Discussion by AlexeyGorbachev** [OrchardCMS/OrchardCore#13256] [Q&A]
Hi all! I need to change background image in TheAgencyTheme. Where can I find it?

**Comment by hishamco** [comment]
It's [header-bg.jpg](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/Assets/src/assets/img/header-bg.jpg)

**Reply by AlexeyGorbachev** [reply]
Thank you for your interest in my problem. 
I have found 3 copies of this file in my working folder?
Ways are the following:
OrchardCore\src\OrchardCore.Themes\TheAgencyTheme\Assets\dist\assets\img
OrchardCore\src\OrchardCore.Themes\TheAgencyTheme\Assets\src\assets\img
OrchardCore\src\OrchardCore.Themes\TheAgencyTheme\wwwroot\assets\img

What is the difference between assets in wwwroout src and dist folder?
Also i found that [https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Themes/TheAgencyTheme/wwwroot/Theme.png](url) is the same file. I changed all images to my, and it works.
Also I do not understand why I'm not able to find none of these files in my publish folder. Why? Do all of them are used as embeded sources?

But then I also needed to exchange header-bg.jpg background image one more time. But I was unable to do it. I deleted obj and bin folders, rebuilt the solution and nothing changes.
I deleted OrchardCore\src\OrchardCore.Cms.Web\wwwroot\Default\is-cache folder?
Is this the only place for cache?
